datadog/fwprovider/data_source_datadog_team.go

```diff
@@ -137,7 +137,7 @@ func (d *datadogTeamDataSource) Read(ctx context.Context, req datasource.ReadReq
 	resp.Diagnostics.Append(resp.State.Set(ctx, &state)...)
 }
 
-func (r *datadogTeamDataSource) updateState(state *datadogTeamDataSourceModel, teamData *datadogV2.TeamData) {
+func (r *datadogTeamDataSource) updateState(state *datadogTeamDataSourceModel, teamData *datadogV2.Team) {
```

Comment:
TODO revert [this commit](https://github.com/DataDog/terraform-provider-datadog/pull/2023/commits/64bf150e745e630eb7326b2a57427ad6e0d2efd1) before merging. It's due to an unrelated compilation error

Reply:
We'll either have to leave it in or make a separate PR updating the go sdk version since the teams model change was merged before the downtimes start bugfix PR.

Reply:
Will leave it in.